MusicHub/main/utils.py

```diff
@@ -103,3 +108,29 @@ def create_or_return_user(backend, response, *args, **kwargs):
 def get_random_string(length):
     letters = string.ascii_lowercase
     return "".join(random.choice(letters) for i in range(length))
+
+
+class LargeResultsSetPagination(PageNumberPagination):
+    """
+    Overrides default pagination class provided in settings.py,
+    inside REST_FRAMEWORK dictionary.
+    """
+
+    page_size = 40
+    page_size_query_parameter = "page_size"
+    max_page_size = 40
+
+
+def format_to_minutes(sec):
+    minutes = str(sec // 60)
+    seconds = str(sec % 60)
+    return minutes + ":" + seconds
+
+
+def delete_generated_files_testing_tracks():
```

Comment:
looks like it's needed only for the tests. If so let's move it to "test's utils"